attestation-service/verifier/src/snp/mod.rs

```diff
@@ -209,21 +220,33 @@ fn verify_signature(cert: &X509, issuer: &X509, name: &str) -> Result<()> {
         .ok_or_else(|| anyhow!("Invalid {name} signature"))
 }
 
-fn verify_cert_chain(cert_chain: &[CertTableEntry], ask: &X509, ark: &X509) -> Result<X509> {
-    let raw_vcek = cert_chain
+fn verify_cert_chain(
+    cert_chain: &[CertTableEntry],
+    ask: &X509,
+    ark: &X509,
+    asvk: &X509,
+) -> Result<X509> {
+    let raw_vek = cert_chain
```

Comment:
Doesn't VEK refer to VM Encryption Key? Maybe call this a signing key or something that could refer to vcek or vlek.